tools/RAiDER/statsPlot.py

```diff
@@ -351,6 +369,11 @@ def resid(x, d, v, m):
                                    loss='soft_l1', f_scale=0.1,
                                    args=(d, v, model))
 
+        # convert range to specified output unit
+        # TODO temp fix first multiply by 1,010,000 m as crude conversion from degrees to km
```

Comment:
the comment is incorrect it is mixing conversion from degree to m and km.

Reply:
Fixed